chain/chain/src/types.rs

```diff
@@ -380,6 +380,40 @@ impl BlockApproval {
     }
 }
 
+/// Block approval by other block producers.
```

Comment:
comment from wrong place?

Reply:
I think so. Seems like the side-effect of merging with staging.